editor/tasks/leiningen/init.clj

```diff
@@ -66,7 +66,7 @@
 (def init-tasks
   [["clean"]
    ["local-jars"]
-   ["builtins"]
+   ; ["builtins"] builtins is in bob which we install in `local-jars` step
```

Comment:
This step downloads builtins.zip. But we already have `builtins` in `bob.jar` which we download in `local-jars` (prev step)